plugins/fileservices/fileservices.cpp

```diff
@@ -2855,6 +2855,23 @@ static bool build_dfuplus_globals(int argc, const char *argv[], IProperties * gl
         if (strchr(argv[i],'='))
             globals->loadProp(argv[i]);
 
+    // if server matches the exact string "eclservices" we can look up the settings for that service
+    const char *server = globals->queryProp("server");
+    if (!isEmptyString(server) && streq(server, "eclservices"))
```

Comment:
question: perhaps instead, this should be the default behaviour if no "server" is provided?
i.e. auto lookup the local service url and local secret, i.e. eclservices is implicit.

Reply:
Yes, agreed.  The thought had crossed my mind.  I think since the implicit server value would be handled in fileservices.DfuPlusExec() and not the command line dfuplus executable it will be fine.  I'll change it.

Reply:
@jakesmith I've updated as discussed, can you check?